01-Setup and getting started/README.md

```diff
@@ -57,6 +57,11 @@ Let's start!
   If the PSVersion is older than 5.0, update PowerShell from [here](https://www.microsoft.com/en-us/download/details.aspx?id=54616).  
 
       ![PowerShell Version](../Images/00_PS_Version.png)
+    
+ 1. c.  PowerShell language mode FullLanguage: </br>
```

Comment:
Somehow if you to go file view, this comes as '2.c' instead of '1.c'

Reply:
We need to add this text in 'method B' as well.

Reply:
That was weird, some indentation related issue. Fixed, Thanks for the catch !